src/graph_compute/phase_scheduling.h

```diff
@@ -20,13 +19,23 @@ typedef struct chunkdata_t chunkdata_t;
 
 struct scheddata_t {
   vid_t * dependentEdges;
```

Comment:
The one in scheddata holds the root of the dependent edges array, whereas the one in the sched_t points into that array on a per-vertex basis.